### PR TITLE
Add `.env` info to docker-compose guide

### PR DESCRIPTION
**Pull request by NickCrews**
Helps avoid (but doesn't solve) https://github.com/directus/directus/issues/20348

### CODE REVIEW

File: docs/self-hosted/docker-guide.md [hunk 1]
```diff
@@ -132,6 +132,9 @@ services:
       # Make sure to set this in production
       # (see https://docs.directus.io/self-hosted/config-options#general)
       # PUBLIC_URL: "https://directus.example.com"
+    # or, if your environment variables are defined in a `.venv` file:
+    env_file:
+      - .env
```
Comment (br41nslug) [inline docs/self-hosted/docker-guide.md:137]:
```suggestion
    # env_file:
    #  - .env
```

Lets keep that as comments for those that want to use an `.env` file.

**Comment by paescuj** [comment]
Could you give some more context how this is related to #20348?
Also isn't it supposed to be `.env` instead of `.venv`?

**Comment by NickCrews** [comment]
Oops yeah it is supposed to be .env not .venv. I'm coming from the python world where I type venv a lot more than env so that was just habit.

I was defining all my environment variables in a .env file as the documentation guides suggest, but then these aren't by default  Getting passed into the container when using docker compose. Therefore, the app didn't have the needed variables and was crashing as described in that issue.

**Comment by paescuj** [comment]
> Oops yeah it is supposed to be .env not .venv. I'm coming from the python world where I type venv a lot more than env so that was just habit.

Haha that's what I thought 😄

_Habit is the sixth sense that dominates the other five._
_- Arabian Proverb_

I was defining all my environment variables in a .env file as the documentation guides suggest, but then these aren't by default Getting passed into the container when using docker compose. Therefore, the app didn't have the needed variables and was crashing as described in that issue.

I see, thanks! It's worth adding this as a hint, IMO 👍

**Review by paescuj — APPROVED** [review]
Thanks @NickCrews!

**Comment by NickCrews** [comment]
I'm curious, if I was using one of the other recommended methods of storing env cars such as in a .js file, how would this need to work? Pass in the CONFIG_PATH, and then also have that file mounted as a volume? I personally am not needing that, but I could see others using one of those other methods and getting equally lost.

**Comment by paescuj** [comment]
> I'm curious, if I was using one of the other recommended methods of storing env cars such as in a .js file, how would this need to work? Pass in the CONFIG_PATH, and then also have that file mounted as a volume?

Exactly, that's it 👍

> I personally am not needing that, but I could see others using one of those other methods and getting equally lost.

Happy to track this in an issue → https://github.com/directus/directus/issues/new?assignees=&labels=Documentation&projects=&template=docs-suggestions.yml

**Pull request merged**